### ISSUE TITLE
Suppress `generator` error on panic

### ISSUE THREAD

**Issue by jamesbornholt** [awslabs/shuttle#48]
When a thread panics in a Shuttle test, [`generator` produces an error](https://github.com/Xudong-Huang/generator-rs/blob/master/src/gen_impl.rs#L558) that gets printed to stdout via `tracing`. This error is a red herring: it mentions `generator::gen_impl` and so suggests that maybe something went wrong in that code, when really it's just that a Shuttle thread panicked and that panic is being propagated. It would be nice to suppress this error so we don't lead users down the wrong debugging path.

**Comment by bkragl** [comment]
+1

I have a Shuttle test running inside a proptest, where a failing execution aborts before showing the input used by proptest (and probably even before proptest tries to minimize).

After the stack backtrace the output finishes as follows:
```
Aug 12 11:16:32.499 ERROR generator::gen_impl: set panicked inside generator    
thread panicked while panicking. aborting.
error: test failed, to rerun pass '-p foo --lib'

Caused by:
  process didn't exit successfully: `/path/to/foo-7e20baac4e3ab4e7 'foo:bar' --exact --nocapture` (signal: 4, SIGILL: illegal instruction)
```

Will suppressing the generator error resolve the nested panicking and SIGILL?

**Comment by jamesbornholt** [comment]
No, the generator error is only an error message printed when the Shuttle thread is panicking -- it doesn't actually create its own panic. Nested panics like this usually happen because a `Drop` handler panics during stack unwinding. We could consider creating an option to disable stack unwinding after a panic (by leaking the generator threads) to make minimization work.
